pkg/ccl/backupccl/full_cluster_backup_restore_test.go

```diff
@@ -133,11 +132,7 @@ CREATE TABLE data2.foo (a int);
 
 	// Setup the system systemTablesToVerify to ensure that they are copied to the new cluster.
 	// Populate system.users.
-	numUsers := 1000
-	if util.RaceEnabled {
-		numUsers = 10
-	}
-	for i := 0; i < numUsers; i++ {
+	for i := 0; i < 10; i++ {
```

Comment:
Maybe I'm missing something, what did you do to prevent 10 schema changes from ocurring? how did this wrap the 10 user additions into a single txn?